aep/general/0134/aep.md.j2

```diff
@@ -90,12 +90,12 @@ message UpdateBookRequest {
 
 - A `path` field **must** be included.
   - The field **must** be [annotated as required](/field-behavior-documentation).
-  - The field **must** identify the [resource type](/resource-types) that it
```

Comment:
I think this link still works - it's just routing to the redirect to aep-4 (which allows us to alias to new AEPs if there's new versions): https://aep.dev/resource-types.

that said the site-generator isn't great with this - it just redirects to aep.dev/4 right now, when it should allow aep.dev/resource-types to be a valid slug.

Is this something handled as part of your rewrite @rambleraptor?

Reply:
I'm sure you're right. I very nearly got carried away normalizing all the link formats, and the [aep-123] format is more prevalent. This change must have slipped through when I restrained my zeal. Would you like me to change it back?

Reply:
Can you confirm if this is considered blocking? The merge won't proceed while this conversation is open.

Reply:
not blocking. Sorry about the delay there!